pkg/commands/secretstore/doc.go

```diff
@@ -0,0 +1,5 @@
+// Package secretstore contains commands to inspect and manipulate Fastly edge
+// secret stores.
+//
+// https://developer.fastly.com/reference/api/secret-store/
```

Comment:
The documentation is marked as 'unlisted' so customers won't be able to find the link themselves unless they happen to check this repository code. We should consider updating the internal API documentation repo to expose this endpoint so customers can find the API information (I see the endpoint is listed as 'beta' so that's fine).

Reply:
Yes, the documentation will be switched over from 'unlisted' soon.